### PR TITLE
README polishes.

### PR DESCRIPTION
**Pull request by concretevitamin**
Tested
- rendered locally

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,18 +8,25 @@
 ![pytest](https://github.com/skypilot-org/skypilot/actions/workflows/pytest.yml/badge.svg)
 [![Documentation Status](https://readthedocs.org/projects/skypilot/badge/?version=latest)](https://skypilot.readthedocs.io/en/latest/?badge=latest)
 
-SkyPilot is a framework for easily running machine learning[^1] workloads on any cloud through a unified interface. No knowledge of cloud offerings is required or expected – you simply define the workload and its resource requirements, and SkyPilot will automatically execute it on AWS, Google Cloud Platform or Microsoft Azure.
+SkyPilot is a framework for easily running machine learning workloads[^1] on any cloud. 
 
 
-### Key features
+Use the clouds **easily** and with **low cost**, all without needing to learn cloud infra.
```
Comment (romilbhardwaj) [inline README.md:14]: *at low cost

File: README.md [hunk 2]
```diff
@@ -8,18 +8,25 @@
...
+* **Easily manage jobs** across multiple clusters
+* **Automatic fail-over** to find scarce resources (GPUs) across regions and clouds
 * **Store datasets on the cloud** and access them like you would on a local file system 
-* **Easily manage job queues** across multiple clusters
+* **No cloud lock-in** – seamlessly run your code across different cloud providers (AWS, Azure or GCP)
 
+_Cost saving_
+* Run jobs on **spot instances** with **automatic recovery** from preemptions
+* Hands-free life cycle: **automatically stop idle clusters**
+* One-click use of **TPUs**, for high-performance, cost-effective training
```
Comment (romilbhardwaj) [inline README.md:28]: Not sure if TPUs are cheaper (may be [more expensive for some models](https://indico.cern.ch/event/1015407/contributions/4261881/attachments/2209662/3739396/20210316_IML_TPUvsGPU.pdf), [cheaper for others](https://eng.snap.com/en-US/training-models-with-tpus)). Maybe move to ease of use heading?

File: README.md [hunk 3]
```diff
@@ -8,18 +8,25 @@
...
+* Automatically benchmark and find the best hardware for your job
```
Comment (romilbhardwaj) [inline README.md:29]: If we're putting this under cost saving, maybe better to say:
* Automatically benchmark and find cost-effective hardware for your job

File: docs/source/conf.py [hunk 4]
```diff
@@ -13,7 +13,7 @@
 copyright = '2022, SkyPilot Team'
 author = 'the SkyPilot authors'
 
-release = '0.1'
+release = '0.1.0'
```
Comment (romilbhardwaj) [inline docs/source/conf.py:16]: We're at 0.1.1 on pypi, should we use that here?

**Review by romilbhardwaj — APPROVED** [review]
Looks great!

File: examples/README.md [hunk 5]
```diff
@@ -21,11 +21,11 @@ Machine learning examples:
...
 
 General examples:
 
 1. [**`multi_echo.py`**](./multi_echo.py): Launch and schedule hundreds of bash commands in the cloud, with configurable resources.  Similar to grid search.
 
-## TODO: non-runnable apps
-1. [**`timm_app.py`**](./timm_app.py): the [PyTorch image models (timm)](https://github.com/rwightman/pytorch-image-models) package.
-  - Not runnable due to requiring ImageNet images, not tfrecords; consider using https://github.com/mlcommons/inference/blob/master/vision/classification_and_detection/tools/make_fake_imagenet.sh.
+
+...and much more!
```
[the remark this answers is not in the document]
Comment (concretevitamin) [reply examples/README.md]: Added more examples and kept "many more".

File: README.md (hunk 1 above)
Comment (concretevitamin) [reply README.md]: Reworded to "cost effectively"

File: README.md (hunk 2 above)
Comment (concretevitamin) [reply README.md:25]: I believe this is increasingly the case and propose we keep it.  For example, Google internally all but abandons GPUs, from what we can tell on recent project releases.

File: README.md (hunk 3 above)
Comment (concretevitamin) [reply README.md]: Changed to "cheapest" now, to avoid "cost effective" showing up in 2 successive items.

File: docs/source/conf.py (hunk 4 above)
Comment (concretevitamin) [reply docs/source/conf.py]: Done, good catch!

**Comment by concretevitamin** [comment]
@romilbhardwaj many changes, PTAL.

**Review by romilbhardwaj — APPROVED** [review]
Awesome! Thanks for making the docs/readme crisper!

**Pull request merged**